src/shared/database/index.ts

```diff
@@ -37,14 +37,11 @@ class Pindexer {
 
   async summary(window: DurationWindow, baseAsset: AssetId, quoteAsset: AssetId) {
     return this.db
-      .selectFrom('dex_ex_pairs_summary as s')
-      .leftJoin('dex_ex_price_charts as c', join =>
-        join.onRef('c.asset_start', '=', 's.asset_start').onRef('c.asset_end', '=', 's.asset_end'),
```

Comment:
The low/high values in the charts table does not map to the summary quite as expected:

> and there's a difference between the 1d candlestick, which is "what has happened from midnight UTC until now, today" vs "what was the lowest price between now and 24h ago" - @cronokirby 

Will simplify the query and update the schema when low/high is available in the summary table